bin/pygrb/pycbc_grb_inj_combiner

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 Duncan Macleod
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""Combine triggers from a splitbank GRB run
+"""
+
+from __future__ import (division, print_function)
+
+import argparse
+import os
+import re
+from collections import defaultdict
+from math import (pi as PI, radians)
+
+import numpy
+
+import tqdm
+
+import h5py
+
+from astropy.constants import M_sun
+
+from pycbc import __version__
+
+MSUN_SI = M_sun.si.value
+
+__author__ = "Duncan Macleod <duncan.macleod@ligo.org>"
+
+TQDM_BAR_FORMAT = ("{desc}: |{bar}| "
+                   "{n_fmt}/{total_fmt} {unit} ({percentage:3.0f}%) "
+                   "[{elapsed} | ETA {remaining}]")
+TQDM_KW = {
+    "ascii": " -=#",
+    "bar_format": TQDM_BAR_FORMAT,
+    "smoothing": 0.05,
+}
+
+INJSTRING_REGEX = re.compile(r"_[A-Z]*(\d+)(INJ)_(FOUND|MISSED)")
+
+# -- utilties -----------------------------------
+
+
+def use_file(filename, inclination):
+    """Returns `True` if this file can be used to analysis the given inclination
+    """
+    match = INJSTRING_REGEX.search(filename)
+    if match and float(match.groups()[0]) >= inclination:
+        return True
+    return False
+
+
+def theta(h5injgroup):
+    """Calculate the theta angle column for an injection group
```

Comment:
I think there's better versions of this function in LAL (including the non-leading terms in L). We might want to switch to a better function if there is one. Worth checking with Seb/Patricia/Harald, but doesn't hold up merging this.

Reply:
Would be nice to have the conversion function in here though:
https://github.com/gwastro/pycbc/blob/master/pycbc/conversions.py

Reply:
I am not confident that I understand the function and its use cases properly enough to translate this into that module. I would rather leave it here for now, and pass the buck to other pycbc devs to move this upstream and modify it to fit appropriately.